README.md

```diff
@@ -113,6 +113,8 @@ If you are not cloning this repo, make sure your dependencies versions are the s
 
 [Installing node.js on Windows](https://treehouse.github.io/installation-guides/windows/node-windows.html)
 
+[Installing bot on a Raspberry](https://github.com/Bacon-Fixation/Master-Bot/blob/pi/Pi_Instructions.md)
```

Comment:
just a Temp link it works but would probably need to be changed to  `https://github.com/galnir/Master-Bot/blob/Pi_Instructions.md`